# Web-based 'Omics' tools which you may need to know when doing bioinformatics (from 2009)

## 1) PrognoScan (2009)
http://dna00.bio.kyutech.ac.jp/PrognoScan/

https://bio.tools/PrognoScan

Exploring the relation/link between gene expression and patient progonsis such as overall survival (OS) and disease free survival (DFS) across a large collection of publicly available cancer microarray datasets.
 
This platform provides a database including a large collection of publically available cancer mircroarray dataset with clinical annotation as well as a tool for evaluating the biological relationship between gene expression and prognosis.

Read the paper from this link: "PrognoScan: a new database for meta-analysis of the prognostic value of genes", https://bmcmedgenomics.biomedcentral.com/articles/10.1186/1755-8794-2-18

------------------------------------

## 2) GOBO (2011)
http://co.bmc.lu.se/gobo/

Assessment of gene expression, identification of co-expressed genes and association with outcome for single genes, gene sets or gene signatures in an 1881-sample breast cancer data set.

------------------------------------

## 3) bc-GeneExMiner (2011)
http://bcgenex.centregauducheau.fr/BC-GEM/GEM-Accueil.php?js=1

breast cancer Gene-Expression Miner v4.0 is a statistical analysis and mining tool with the following purposes: 
"expression", "prognosis" and "correlation" 

------------------------------------

## 4) RecurrenceOnline (2012)
http://www.recurrenceonline.com/

Recurrence Online is an online diagnostic service capable to predict:
survival [risk of relapse and recurrence score], response to hormonal treatment [ER status] and response to targeted therapy [HER2 status]

------------------------------------

## 5) TMA Navigator (2013)
http://www.tmanavigator.org/

TMA Navigator is a system for analysing tissue microarray (TMA) data. There are several tools available for data exploration, network inference and survival analysis. 

Example datasets demonstrate the TMA Navigator workflow without having to upload one's own data:

Breast Cancer 1 (AQUA): 128 breast cancer patients, 9 proteins, Breast Cancer 2 (AQUA): 92 breast cancer patients, 16 proteins, Breast Cancer 3 (IHC, 5 levels): 122 breast cancer patients, 4 proteins, Breast Cancer 3 (IHC quickscore): 122 breast cancer patients, 4 proteins

------------------------------------

## 6) KMPlot (2013)
http://kmplot.com/analysis/

The Kaplan Meier plotter is capable to assess the effect of 54,675 genes on survival using 10,461 cancer samples. These include 5,143 breast, 1,816 ovarian, 2,437 lung and 1,065 gastric cancer patients with a mean follow-up of 69 / 40 / 49 / 33 months. Primary purpose of the tool is a meta-analysis based biomarker assessment.

------------------------------------

## 7) Cutoff Finder (2013)
http://molpath.charite.de/cutoff/

Cutoff Finder is a web-based application for the determination of cutoff points in molecular data. Often, molecular data such as gene expression or protein expression data are represented by continuous or at least ordinal variables. In order to translate these variables into a clinical decision, it is necessary to determine a cutoff point and to stratify patients into two groups, each of which requiring a different kind of treatment. Cutoff Finder offers a multitude of methods for cutoff determination and plots.

------------------------------------

## 8) PROGgeneV2 (2013)
http://watson.compbio.iupui.edu/chirayu/proggene/database/index.php

PROGgeneV2 has several advancements over its predecessor. One important feature is enabling study of prognostic implications of gene-gene expression ratio. they have also added the capability to upload custom datasets. Users can upload their own gene expression datasets to the tool and perform survival analysis on genes of interest. 

------------------------------------

## 9) SurvExpress (2013)
http://bioinformatica.mty.itesm.mx:8080/Biomatec/SurvivaX.jsp

A cancer-wide gene expression database with clinical outcomes and a web-based tool that provides survival analysis and risk assessment of cancer datasets. The main input of SurvExpress is only the biomarker gene list. We generated a cancer database collecting more than 20,000 samples and 130 datasets with censored clinical information covering tumors over 20 tissues.

------------------------------------

## 10) bioportal (2013)
A portal containing 150 cancer studies:  
http://www.cbioportal.org/, http://www.cbioportal.org/cgds_r.jsp

------------------------------------

## 11) Tumor Portal (2014)
http://dev.tumorportal.org/charts/clinical

The site includes graphical displays of the mutations in each of the 18,388 genes studied in this paper: "Discovery and saturation analysis of cancer genes across 21 tumour types". The site also includes tables of mutational data for each significant gene and Qâ€“Q plots for each statistical test.

------------------------------------

## 12) MIMIC - MInimal Methylation Classifier (2017) - 
### Ask your network administrator to open the port 3838 in your firewall in case that your browser can't open the landing page 
http://medulloblastomadiagnostics.ncl.ac.uk:3838/landing/

## 13) wANNOVAR
ANNOVAR is a rapid, efficient tool to annotate functional consequences of genetic variation from high-throughput sequencing data. wANNOVAR provides easy and intuitive web-based access to the most popular functionalities of the ANNOVAR software

http://wannovar.wglab.org/

## 14) DeSigN (2016) 

Connecting gene expression with therapeutics for drug repurposing and development
This web-based tool aims to predict drug efficacy against cancer cell lines using gene expression patterns. The algorithm correlates phenotype-specific gene signatures derived from differentially expressed genes with pre-defined gene expression profiles associated with drug response data (IC50) from 140 drugs

http://design.cancerresearch.my/

------------------------------------
# Web-based Clustering Software 

## 15) A web tool for visualizing clustering of multivariate data (BETA) 
This web tool allows users to upload their own data and easily create Principal Component Analysis (PCA) plots and heatmaps. Data can be uploaded as a file or by copy-pasteing it to the text box.

https://biit.cs.ut.ee/clustvis/

------------------------------------
## 16) Number of software analyses for gene expression microarray data
https://www.arabidopsis.org/portals/expression/microarray/microarraySoftwareV2.jsp

------------------------------------
## 17) Next-Generation Clustered Heat Maps (NG-CHMs) covering multiple tumor types and multiple data types profiled by The Cancer Genome Atlas (TCGA) Project.
http://bioinformatics.mdanderson.org/TCGA/NGCHMPortal/

------------------------------------
## 18) iAtlas: Interactive portal for immuno-oncology research
iAtlas is an interactive web-based platform and set of analytic tools for studying interactions between tumors and the immune microenvironment.

https://www.cri-iatlas.org/

## 19) TCGA Batch Effects Tool

This website is designed to help assess, diagnose and correct for any batch effects in TCGA data. It first allows the user to assess and quantify the presence of any batch effects via algorithms such as Hierarchical Clustering and Principal Component Analysis. The results from these algorithms are presented graphically as both simple and interactive diagrams. If significant batch effects is observed in the data, the user then has the option of downloading data that has been computationally corrected using methods such as Empirical Bayes (aka. ComBat), Median Polish and ANOVA.

Because the samples are processed in batches rather than all at once, the data can be vulnerable to systematic noise such as batch effects (unwanted variation between batches) and trend effects (unwanted variation over time), which can lead to misleading analysis results. 

http://bioinformatics.mdanderson.org/main/TCGABatchEffects:Overview
